src/Database/Query.php

```diff
@@ -226,6 +226,37 @@ public function execute()
     }
 
     /**
+     * Executes the SQL of this query and immediately closes the statement before returning the row count of records
+     * changed.
+     *
+     * This method can be used with UPDATE and DELETE queries, but is not recommended for SELECT queries and is not
+     * used to count records.
+     *
+     * ## Example
+     *
+     * ```
+     * $rowCount = $query->update('articles')
+     *                 ->set(['published'=>true])
+     *                 ->where(['published'=>false])
+     *                 ->executeAndClose();
+     * ```
+     *
+     * The above example will change the published column to true for all false records, and return the number of
+     * records that were updated.
+     *
+     * @return int
+     */
+    public function executeAndClose()
```

Comment:
`public function finishHim()` @ https://www.youtube.com/watch?v=SV6JYR9PwIY

Well this is what it does? `executeAndCloseStatementThenMaybeReturnCount()`

`executeReturnCountClose()` maybe but I do like `executeAndClose()` tbh